deploy/manifests/controller/helm/retina/values.yaml

```
@@ -113,6 +120,9 @@ resources:
   limits:
     memory: "300Mi"
     cpu: "500m"
+  requests:
+    memory: "300Mi"
+    cpu: "100m"
```

This is a behavior change, as the Pods will now deploy with a QoS class of "Burstable" instead of the [previous class of "Guaranteed"](https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed) since Requests = Limits when Requests aren't specified.

> [Note: If you specify a limit for a resource, but do not specify any request, and no admission-time mechanism has applied a default request for that resource, then Kubernetes copies the limit you specified and uses it as the requested value for the resource.](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits)

Can we keep them equal but configurable, to maintain the "Guaranteed" QoS by default?
```suggestion
    cpu: "500m"
```